packages/calcite-components/src/components/link/link.tsx

```diff
@@ -114,15 +114,19 @@ export class Link implements InteractiveComponent, LoadableComponent {
           This works around that issue for now.
           */
             download={
-              Tag === "a" ? (download === true || download === "" ? "" : download || null) : null
+              childElType === "a"
```

Comment:
As per line 103 in this file (`const Tag = childElType;`), Tag and childElType variables have the same value in Stencil - thus this change should have no runtime impact.
However, in Lit, `Tag` function has a bit different type to comply with Lit's dynamic tag name syntax.

See Lumina's docs on JSX -> Dynamic tag names for more details